src/public/admin/logs/styles.css

```
@@ -275,30 +284,70 @@
-  .logs-th {
-    padding: 8px 12px;
-    font-size: 10px;
+
+  .logs-table thead {
+    position: absolute;
+    width: 1px;
+    height: 1px;
+    padding: 0;
+    margin: -1px;
+    overflow: hidden;
+    clip: rect(0, 0, 0, 0);
```

Stylelint (17.10.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)